server_admin/topics/clients/client-oidc.adoc

```diff
@@ -149,6 +149,21 @@ WARNING:
 None of the keycloak client adapters currently support holder-of-key token verification.
 Instead, keycloak adapters currently treat access and refresh tokens as bearer tokens.
 
+[[_proof-key-for-code-exchange]]
+*Proof Key for Code Exchange (PKCE)*
+
+PKCE prevents an attacker who steals an authorization code issued to a legitimate client from receiving the tokens in return to their stealing authorization code.
```

Comment:
I suggest wording it this way:
When an attacker steals an authorization code that was issued to a legitimate client, PKCE prevents the attacker from receiving the tokens that apply to that code.

Reply:
Thank you. I'll revise the sentence as you suggested.